lib/actions.js

```diff
@@ -201,6 +201,14 @@ function reqInstance(method, path, data) {
     })
 }
 
+function resolveUrlWithParams(url, queryParams) {
+  // take in queryParams as object, and transform into ?key=value&key2=value2
```

Comment:
You should be able to just use the built-in URLSearchParams helper class:
```js
const qs = new URLSearchParams(queryParams);
return [url, qs.toString()].join("?");
```

https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams

Reply:
Nice. Ok I'll update

Reply:
`querystring` is another alternative:
```js
const qs = querystring.stringify(queryParams);
```

Reply:
That one was for myself haha. Always looking for an excuse to build a clean reduce implementation lol